addons/crm/models/res_partner.py

```diff
@@ -50,12 +50,8 @@ def _compute_meeting_count(self):
             partner.meeting_count = len(partner.meeting_ids)
 
     def schedule_meeting(self):
-        partner_ids = self.ids
-        partner_ids.append(self.env.user.partner_id.id)
         action = self.env.ref('calendar.action_calendar_event').read()[0]
-        action['context'] = {
```

Comment:
I'm not sure how this relates. `default_partner_ids` gives the default values when creating a new meeting, it doesn't impact the list of meetings displayed when clicking on the stat button.